program/Cargo.toml

```diff
@@ -1,8 +1,28 @@
 [package]
 name = "solana-feature-gate-program"
 version = "0.1.0"
+description = "Solana Feature Gate Program"
+authors = ["Anza Technology Maintainers <maintainers@anza.xyz>"]
+repository = "https://github.com/solana-program/feature-gate"
+readme = "./README.md"
+license-file = "../LICENSE"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[package.metadata.solana]
+program-id = "Feature111111111111111111111111111111111111"
+
+[features]
+bpf-entrypoint = []
+test-sbf = []
 
 [dependencies]
+num_enum = "0.7.2"
+solana-program = "1.18.2"
+spl-program-error = "0.3.1"
```

Comment:
Should this be updated to 0.4?

Reply:
It can be, yeah! But for now I left it as it's written in SPL. Fwiw, I planned to bump each of these programs to the newer `0.4` if they weren't already.

Reply:
I only bring it up because:

https://github.com/solana-labs/solana-program-library/blob/c18ae71a499aee51480bde4375728dc81939fef7/feature-gate/program/Cargo.toml#L17

Reply:
Ohhh you caught me. I must have copied that from one of the other BPF guys in this org. Bumped it anyway!